lib/msal-common/apiReview/msal-common.api.md

```diff
@@ -1466,7 +1459,7 @@ export type ClientConfiguration = {
     authOptions: AuthOptions;
     systemOptions?: SystemOptions;
     loggerOptions?: LoggerOptions;
-    cacheOptions?: CacheOptions;
+    cacheOptions?: undefined;
```

Comment:
This shouldn't show up here at all if it's only typed as undefined. Can you double check this?

Reply:
Interestingly, even after removing cacheOptions altogether, this still remains the same and passes tests.

Reply:
Did you push the changes? I still see it below

Reply:
Weird, okay it's fixed now.